docs/PORTING.md

```diff
@@ -168,23 +150,28 @@ int     flash_area_id_from_multi_image_slot(int image_index, int slot);
 int     flash_area_id_to_multi_image_slot(int image_index, int area_id);
 ```
 
-**Note:** As of writing, it is possible that mcuboot will open a flash area multiple times simultaneously (through nested calls to `flash_area_open`). As a result, mcuboot may call `flash_area_close` on a flash area that is still opened by another part of mcuboot. As a workaround when porting, it may be necessary to implement a counter of the number of times a given flash area has been opened by mcuboot. The `flash_area_close` implementation should only fully deinitialize the underlying flash area when the open counter is decremented to 0. See [this GitHub PR](https://github.com/mcu-tools/mcuboot/pull/894/) for a more detailed discussion.
+---
+**Note**
+
+As of writing, it is possible that MCUboot will open a flash area multiple times simultaneously (through nested calls to `flash_area_open`).
+As a result, MCUboot may call `flash_area_close` on a flash area that is still opened by another part of MCUboot.
+As a workaround when porting, it may be necessary to implement a counter of the number of times a given flash area has been opened by MCUboot.
+The `flash_area_close` implementation should only fully deinitialize the underlying flash area when the open counter is decremented to 0.
+See [this GitHub PR](https://github.com/mcu-tools/mcuboot/pull/894/) for a more detailed discussion.
+
+---
 
 ## Memory management for mbed TLS
```

Comment:
```suggestion
## Memory management for Mbed TLS
```
According to the most recent website, `Mbed TLS` is now stylized with a capital M.

Reply:
Could you link it here? I found only this: https://tls.mbed.org/ and it seems like they are using mbed TLS way more than Mbed TLS, especially in their docs: https://tls.mbed.org/api/

Reply:
The https://tls.mbed.org/ website has been informing for some time the following notice:
![image](https://user-images.githubusercontent.com/38959758/132706337-05a6d144-15ec-49f9-9f61-489307ba5355.png)

And every reference to Mbed TLS on the [TrustedFirmware](https://www.trustedfirmware.org/) website contains the capital M.

Reply:
The [Mbed TLS GitHub repository](https://github.com/ARMmbed/mbedtls) also stylizes with the capital M.
It was changed from `mbed TLS` to `Mbed TLS` in this PR: https://github.com/ARMmbed/mbedtls/pull/1159

Reply:
I see, You are right. Somehow I managed to not notice what's written in the note as I was just checking for instances of mbed or Mbed :D Will fix this in the next commit.

Reply:
fixed